packages/utils/telemetry-utils/api-report/telemetry-utils.api.md

```diff
@@ -219,7 +219,11 @@ export interface ITelemetryGenericEventExt extends ITelemetryPropertiesExt {
     // (undocumented)
     category?: TelemetryEventCategory;
     // (undocumented)
+    continuityID?: string | undefined;
+    // (undocumented)
     eventName: string;
+    // (undocumented)
+    sessionID?: string | undefined;
```

Comment:
I imagine the PR build will report an unexpected change in this file. Need to rebuild this package to undo the changes to its public API surface.